.openapi/kas-fleet-manager.yaml

```diff
@@ -1267,6 +1267,7 @@ components:
               - $ref: "#/components/schemas/RegionCapacityListItem"
       required:
         - enabled
+        - supported_instance_types
```

Comment:
@gowriswarupk Depends on the timing we should manually remove this change or look for merge of:
https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager